assimilation_code/modules/io/direct_netcdf_mod.f90

```diff
@@ -916,8 +954,7 @@ subroutine read_transpose_single_task(state_ens_handle, name_handle, domain, dar
    iend = istart + block_size -1
 
    if (query_read_copy(name_handle, copy)) then
-      call read_variables(ncfile, vector, 1, get_num_variables(domain), domain)
-      ! close netcdf file
+      call read_variables(ncfile, vector, start_var, get_num_variables(domain), domain)
```

Comment:
why was the code starting at 1 before and now respects start_var?  is this going to break something?  it is counting on all the places this is called from to have the right start_var value, which it apparently didn't matter before.  

anything in this file is more than just CLM changes - this is read/write for all of dart, every model.  i'd like to understand this change better before agreeing to it.